Source/Modules/go.cxx

```diff
@@ -10,6 +10,7 @@
 #include "swigmod.h"
 #include "cparse.h"
 #include <ctype.h>
+#include <stdint.h>
```

Comment:
Is it possible to introduce md5 or sha1 into Swig core?

What I really need is a way to generate a unique id given
the swig input file, and I expect other language modules
might have similar needs in the future.